src/d3a/models/area/__init__.py

```diff
@@ -266,10 +266,9 @@ def set_events(self, event_list):
     def activate(self, bc=None, current_tick=None, simulation_id=None):
```

Comment:
`bc` is now only a flag to enable of disable the blockchain interface, right? Could you please use a boolean value as default value then? Thanks!

Reply:
A second too late :) let me add to the ticket with the next blockchain updates

Reply:
i was too late. But all good, will change it in my next PR. Thanks!